.github/workflows/bloc_presentation-test.yml

```
@@ -0,0 +1,64 @@
+name: bloc_presentation test
+
+on:
+  push:
+    branches: [master]
+    ignore-tags: ['bloc_presentation-v*']
+    paths:
+      - 'packages/bloc_presentation/**'
+      - .github/workflows/bloc_presentation-test.yml
+  pull_request:
+    branches: [master]
+    paths:
+      - 'packages/bloc_presentation/**'
+      - .github/workflows/bloc_presentation-test.yml
+
+jobs:
+  test:
+    name: Flutter ${{ matrix.version }}
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ['3.3.x', '3.7.x']
+
+    defaults:
+      run:
+        working-directory: packages/bloc_presentation
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Flutter
```

Here "set up" is a verb, not a noun ("setup"), so:

```suggestion
      - name: Set up Flutter
```